### PR TITLE
Feature: Visitor can see different products on a restaurants menu

### PR DESCRIPTION
**Pull request by raptorf1**
### User Story
```
As a visitor,
In order to get an overview of the restaurant products
I would like to be able to see a menu
```

### What changes we propose
* Create database for the dishes to be stored. Settings of the database are ready to support the cartify gem.
* Edits the product controller to display all available dishes in the database.
* Create unit test for the created database.
* Create feature test with cucumber.

### What we have learned
* That in a HAML file the `=` sign makes a ruby code appear in the page, while the `-` makes the code not to appear.
* More HAML syntax.

### Screenshots

Screenshot from 2019-05-03 08-56-27

### CODE REVIEW

File: spec/models/product_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    it { is_expected.to have_db_column :price }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_length_of :name }
+    it { is_expected.to validate_presence_of :description } 
+    it { is_expected.to validate_presence_of :price }
+    it { is_expected.to validate_length_of :price }
+
```
Comment (oliverochman) [inline spec/models/product_spec.rb]: remove empty lines

File: app/models/product.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Product < ApplicationRecord
+    validates :name, :description, :price, presence: true
```
Comment (oliverochman) [inline app/models/product.rb]: refactor this to `validates_presence_of :name, :description, :price`

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/models/product.rb (hunk 2 above)
Comment (Carrosen) [reply app/models/product.rb]: on it

File: spec/models/product_spec.rb (hunk 1 above)
Comment (Carrosen) [reply spec/models/product_spec.rb]: on it

**Review by oliverochman — APPROVED** [review]
Good job guys

**Pull request merged**